Makefile.am

```diff
@@ -715,6 +715,8 @@ dist_noinst_HEADERS = \
     src/util/sss_format.h \
     src/util/sss_pam_data.h \
     src/util/refcount.h \
+    src/util/inotify.h \
```

Comment:
It's already in the list (11 lines below).

Reply:
Fixed